### ISSUE TITLE
Unique uuid by content not by localization.

### ISSUE THREAD

**Issue by IvanWala** [bn-digital/strapi#4]
Hi I want to suggets if is possible give to the gestor the possibility of create an unique uuid by all the localizations of the same content. 

I mean we have a product in spanish, english, french and russian, and we need the same uuid for all the languages. In other firlds we can disallow the i18n. I mean we need to create a link between them. 

Is this possible?

Thanks for your contribution to the comunity.

**Comment by barisx** [comment]
yes. but uuid has own standarts. when we crypte country code or smh about it then add to uuid then it will not be uuid. lets named it `cuuid`.
